service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -64,4 +70,30 @@ private void run2885(CaseDetails caseDetails) {
 
         caseDetails.getData().put("cancelledHearingDetails", caseData.getCancelledHearingDetails());
     }
+
+    private void run2947(CaseDetails caseDetails) {
+        CaseData caseData = getCaseData(caseDetails);
+
+        if (List.of(1602246223743823L, 1611588537917646L).contains(caseData.getId())) {
+            if (isEmpty(caseData.getCancelledHearingDetails())) {
```

Comment:
how it is different from prev migration?
https://github.com/hmcts/fpl-ccd-configuration/pull/2467/files#diff-a5c05dc5ac737cb04f5bcc16602b88786a3197d80b87ff6919ca713bd6b7d015